doc/user-docs/UsingProfiles.md

```diff
@@ -102,17 +113,18 @@ Assuming that you've installed MSYS2 into `C:/msys64`:
     "icon": "C:/msys64/msys2.ico",
     "startingDirectory" : "C:/msys64/home/user"
 }
-````
+```
 
-## Developer Command Prompt for Visual Studio
+## SSH
 
-Assuming that you've installed VS 2019 Professional:
+Starting with Windows 10 1809, you can install the native OpenSSH component. Details can be found [here](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse).
```

Comment:
Come to think about it, WT only runs on 1903 and later. Maybe this is redundant.

Reply:
We should leave this as-is. We don't want to lie about when win32-openssh was first made available to Windows even if it's recommended to install from github directly (more frequent updates) or chocolately. If anything you could add this recommendation. The 8.1 release contains a number of win32 specific bug fixes and features that make the experience better, especially when using Windows Terminal.